sickbeard/webserve.py

```diff
@@ -4478,6 +4478,19 @@ def saveProviders(self, newznab_string='', torrentrss_string='', provider_order=
                 except:
                     curTorrentProvider.confirmed = 0
 
+            if hasattr(curTorrentProvider, 'ranked'):
+                try:
+                    curTorrentProvider.ranked = config.checkbox_to_value(
+                        kwargs[curTorrentProvider.getID() + '_ranked'])
+                except:
+                    curTorrentProvider.ranked = 0
+
+            if hasattr(curTorrentProvider, 'sorting'):
+                try:
+                    curTorrentProvider.sorting = kargs[curTorrentProvider.getID() + '_sorting']
```

Comment:
@miigotu I don't know for sure if this is correct. Can you confirm please? Also please check the latests commits about "sorting"